### PR TITLE
Enabling searching static web app by app name or default host name

### PR DESCRIPTION
**Pull request by XiaoxuHU**
This PR is for enabling searching Static Web App by app name or default host name via Observer

Now in landing page dropdown, we have a new option "Static Web App"
image

If multiple app with same name, it has the same experience as Web App and Container App,
image

In home page, it shows info fetched from Observer
image

### CODE REVIEW

**Comment by XiaoxuHU** [comment]
I will merge this PR after Jeff's change for Observer deployed into production

**Review by SharmaAjay19 — COMMENTED** [review]
Looks alright, assuming that the Runtimehost supports the Microsoft.Web/staticsites resource type. If not, then that change will be needed first.
Edit: It's supported by ARM resource type I see. Looks good XiaoXu!

**Comment by XiaoxuHU** [comment]
> Looks alright, assuming that the Runtimehost supports the Microsoft.Web/staticsites resource type. If not, then that change will be needed first.

**Review by SharmaAjay19 — APPROVED** [review]

File: AngularApp/projects/applens/src/app/modules/dashboard/dashboard-container/dashboard-container.component.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
```
Comment (SharmaAjay19) [inline AngularApp/projects/applens/src/app/modules/dashboard/dashboard-container/dashboard-container.component.ts:2]: Needed?

Comment (XiaoxuHU) [reply AngularApp/projects/applens/src/app/modules/dashboard/dashboard-container/dashboard-container.component.ts]: Removed

**Review by cindylovescoding — APPROVED** [review]

**Pull request merged**